src/public/class-sesamy-content-container.php

```diff
@@ -239,7 +239,7 @@ function( $perk ) {
 						{
 							"type": "<?php echo $pass['period']; ?>",
 							"text": "<?php echo $pass['title']; ?>",
-							"description": "<?php echo $pass['description']; ?>",
+							"description": <?php echo json_encode($pass['description']); ?>,
```

Comment:
So these citation marks aren't needed?

Reply:
Exactly, they are included with the `json_encode` return